src/lib/link.spec.ts

```diff
@@ -1,18 +1,18 @@
-import { sleep } from '@cosmjs/utils';
+import { assert, sleep } from '@cosmjs/utils';
 import test from 'ava';
 
 import { State } from '../codec/ibc/core/channel/v1/channel';
 
 import { prepareChannelHandshake } from './ibcclient';
-import { Link } from './link';
+import { Link, RelayedHeights } from './link';
 import {
   ics20,
-  randomAddress,
   setup,
   simapp,
   TestLogger,
+  transferTokens,
   wasmd,
-} from './testutils.spec';
+} from './testutils';
```

Comment:
@ethanfrey This filename had a `.spec` in it to group it with test files, eg so that it's filtered out of the files published to npm.

Reply:
When I removed the unit tests there and left it as spec, ava complained (errored).

Maybe just add this file to the npmignore file?